README.md

```diff
@@ -1,50 +1,72 @@
-# Enveloop
+# Enveloop Ruby
 
-enveloop-ruby is a ruby wrapper for the Enveloop API. It allows easy use of the API to send messages.
+Hello! 👋 `enveloop-ruby` is a Ruby wrapper for the [Enveloop](https://enveloop.com) API. It simplifies configuring and sending messages (email & SMS/text) via Enveloop inside your Ruby and Ruby on Rails apps.
 
-## Installation
+## What is Enveloop?
 
-Add this line to your application's Gemfile:
+Enveloop is a developer-focused message builder and API that makes it easy to design & send beautiful emails and texts from your app using a simple implementation. You can [sign up for free](https://app.enveloop.com) and be up and running in a couple of minutes!
+
+## Installing the Enveloop Ruby Gem
+
+There are a couple of ways to get started. Using [RubyGems](https://rubygems.org/), you can install it locally with the following command:
+
+```console
+gem install enveloop
+```
+
+Additionally, you can add the following to your application's Gemfile:
 
 ```ruby
 gem 'enveloop'
 ```
 
-And then execute:
+After doing so, you can run the following terminal command:
 
-    $ bundle install
+```console
+bundle install
+```
+
+## Using the Enveloop Gem
 
-Or install it yourself as:
+The Enveloop Gem includes helpful methods to interact with the Enveloop API, including:
 
-    $ gem install enveloop
+* `send_message`
+* `send_raw`
```

Comment:
do we want to advertise this?

Reply:
instead of send raw we might show them how to send HTML without a template

Reply:
I didn't do a write up for it -- per our conversation. It's in the code. But, I am indifferent. We can remove it.

Reply:
Removed `send_raw`. Also, I did leave in, and expand a bit, the discussion around how to use `send_message` and send HTML.

Reply:
awesome .. thank you